Mathlib/Computability/Halting.lean

```diff
@@ -230,10 +230,12 @@ theorem rice₂ (C : Set Code) (H : ∀ cf cg, eval cf = eval cg → (cf ∈ C 
             exact ⟨by infer_instance, Computable.const _⟩ }⟩
 #align computable_pred.rice₂ ComputablePred.rice₂
 
+/-- The Halting problem is recursively enumerable */
```

Comment:
This is a syntax error

Reply:
Apologies I should not have applied the Awaiting Review label until checks were complete—I am looking at the checks. I will be making commits from VS Code going forward